rules/no-boolean-parameters.test.ts

```diff
@@ -1,58 +1,302 @@
-// Use --report-unused-disable-directives to validate errors
+import { ESLintUtils, TSESLint } from "@typescript-eslint/experimental-utils";
 
-/* eslint-disable @typescript-eslint/no-inferrable-types */
-/* eslint-disable @typescript-eslint/no-empty-function */
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const rule = require("./no-boolean-parameters") as TSESLint.RuleModule<
```

Comment:
we could just require a compile step before running the tests - shouldn't be too hard to set up.

other reasons they are in js aren't super valid:
- I think one original rule came from the studio repo where we didn't want to require a compile step
- when writing the triple equals rule I was just lazy because most example code was in JS